### ISSUE TITLE
Unexpected 504 for room

### ISSUE THREAD

**Issue by mrjcleaver** [hipchat/hipchat-rb#157]
Anyone seen this before?

````
Error: #<HipChat::UnknownResponseCode: Unexpected 504 for room `Playlist Builder'>"]
/Volumes/Storage/martincleaver/.chefdk/gem/ruby/2.1.0/gems/hipchat-1.5.2/lib/hipchat/room.rb:142:in `send': Unexpected 504 for room `Playlist Builder' (HipChat::UnknownResponseCode)
````

504 doesn't even appear in https://www.hipchat.com/docs/apiv2/response_codes or https://www.hipchat.com/docs/api/response_codes

I suppose I'll just have to catch such exceptions. Weird.

**Comment by rberrelleza** [comment]
That exception typically happens if something in the backed takes longer than expected to respond back, your safest bet is to add a catch and retry. Is it happening  a lot?

**Comment by mrjcleaver** [comment]
Yeah, I'm getting all sorts of exceptions actually. Eg no such room after creating a room - yet then I see in actuality the room does get created. I was just considering wrapping he HipChat class to make exceptions non-fatal - or throttling somehow. Any advice? Thanks, M.
